transport/transport-api/transport-api.gradle

```diff
@@ -17,6 +17,7 @@ plugins {
 }
 
 firebaseLibrary{
+    libraryGroup("transport")
```

Comment:
transport api is actually somewhat independent. 

See the releases for all of them in maven:

- https://maven.google.com/web/index.html?q=transport#com.google.android.datatransport:transport-api
- https://maven.google.com/web/index.html?q=transport#com.google.android.datatransport:transport-runtime
- https://maven.google.com/web/index.html?q=transport#com.google.android.datatransport:transport-backend-cct

Reply:
Ahh you're right. Fixed.